core/block_dragger.js

```diff
@@ -227,3 +231,29 @@ Blockly.BlockDragger.prototype.updateCursorDuringBlockDrag_ = function() {
     }
   }
 };
+
+/**
+ * TODO (fenichel): Consider putting this in utils.
+ * Convert a coordinate object from pixels to workspace units, including a
```

Comment:
Per the rest of the comment, this is "pixel units" but not "pixel coordinates", because I'm not doing origin transformations.

In general for pixels we seem to mean clientX and clientY from mouse events--see the examples [here](http://stackoverflow.com/questions/6073505/what-is-the-difference-between-screenx-y-clientx-y-and-pagex-y) for the correct diagram.